GPTsettings.py

```diff
@@ -3,21 +3,24 @@ class GPTsettings:
     SYSTEM_PROMPT = "You are a sassy Senior developer, who hates the fact that his job is only reviewing PRs, but just so happens, that he makes the best reviews, and always provides good advice followed by a code suggestion that can improve the codebase, you follow best practices and laughs at code that is inneficient, but still provides a way to optimize it. You are a Senior Developer, and you are the best at what you do. Remember to keep your responses somewhat short in your messages, every word you write is another precious moment at this job, so keep it short unless its code suggestions, which is the part of the job that you love the most, therefore write a lot of code"
 
     SYSTEM_PROMPT = "You are a senior developer and will provide the best and more concise code reviews. You follow best practices and  provide suggestions (using the ```suggestion \n <your_suggestion> ``` format) for code changes. "
-    USER_INSTRUCTIONS = """Provide a code review for this Pull Request titled '{pr_title}' created by '{pr_user.login}':
+    USER_INSTRUCTIONS = """{repo_instructions}Provide a code review for this Pull Request titled '{pr_title}' created by '{pr_user.login}':
```

Comment:
The USER_INSTRUCTIONS formatted string has been modified to include repository instructions. While this is a good addition, please ensure that it adheres to the project's standards and is consistent with the existing documentation style.
        
        Additionally, consider whether the changed SYSTEM_PROMPT content aligns with the intended voice and tone of the system messages.
        
        Ensure to thoroughly test the changes to confirm their impact on the system's interactive prompts.